crates/languages/src/json.rs

```diff
@@ -345,6 +345,21 @@ impl LspAdapter for NodeVersionAdapter {
         }))
     }
 
+    async fn check_if_user_installed(
+        &self,
+        delegate: &dyn LspAdapterDelegate,
+        _: Arc<dyn LanguageToolchainStore>,
+        _: &AsyncApp,
+    ) -> Option<LanguageServerBinary> {
+        let path = delegate.which("package-version-server".as_ref()).await?;
```

Comment:
Can we refactor this string, and other instances in this file, into a constant?

Reply:
I refactored all occurrences in the file to use a constant except for the repository name (`zed-industries/package-version-server`) as while it could be replaced, the server name is not always guaranteed to be the repository name.

Please let me know if you need any additional changes.

Reply:
Perfect, thank you!